src/Network/Probe.php

```diff
@@ -333,7 +333,7 @@ public static function uri($uri, $network = "", $uid = 0, $cache = true) {
 			/// The biggest problem is the avatar picture that could have a reduced image size.
 			/// It should only be updated if the existing picture isn't existing anymore.
 			if (($data['network'] != NETWORK_FEED)
-				AND ($mode == PROBE_NORMAL)
+				AND ($data["network"] != NETWORK_DIASPORA)
```

Comment:
because `$mode` isn't initialized anywhere so the condition would always be false. The code was moved was moved form the the old `function probe_url()` which does have `$mode` as parameter. (https://github.com/friendica/friendica/commit/f5593b19812a854b45a32c93a86f6df45a0cd23d) However, we seem to distinguish between `PROBE_NORMAL` and `PROBE_DIASPORA` so I thought that if the network  is `NETWORK_DIASPORA` we have done a probing to diaspora.
We could also check for `$network != NETWORK_DIASPORA` instead of `$data["network"]` but it seems to me the same.